update_registry_final.py

```
@@ -0,0 +1,34 @@
+import re
+import os
+
+with open('plans/ADR_REGISTRY.md', 'r') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)